### PR TITLE
Fix Xcode 13 build

### PR DESCRIPTION
**Pull request by raptorxcz**
Fix Xcode 13 build

### CODE REVIEW

**Comment by raptorxcz** [comment]
I do not know why Xcode 13 is not available in workflows. Any idea?

**Comment by raptorxcz** [comment]
By documentation https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idruns-on label macos-latest refers to _macOS Catalina 10.15_ instead of _macOS Big Sur 11_.

**Comment by raptorxcz** [comment]
@yonaskolb Can you release after this PR?

**Comment by raptorxcz** [comment]
I seem that issue is in project _antitypical/Result_ itself. Unfortunately I have no idea how to fix this. Could some help me with it?

**Comment by 0111b** [comment]
It looks like it require workaround for Carthage similar to the Xcode 12
https://github.com/Carthage/Carthage/issues/3201

**Comment by mthole** [comment]
@raptorxcz Kudos for getting this going!

**Comment by mthole** [comment]
Green build in https://github.com/yonaskolb/XcodeGen/pull/1130

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 - Fix broken codesign option for bundle dependency [#1104](https://github.com/yonaskolb/XcodeGen/pull/1104) @kateinoigakukun
 - Ensure fileTypes are mapped to JSON value [#1112](https://github.com/yonaskolb/XcodeGen/pull/1112) @namolnad
 - Fix platform filter for package dependecies [#1123](https://github.com/yonaskolb/XcodeGen/pull/1123) @raptorxcz
+- Fix Xcode 13 build [#1127](https://github.com/yonaskolb/XcodeGen/issues/1127) @raptorxcz
```
Comment (freddi-kit) [inline CHANGELOG.md:15]: Please move it to Next Version

**Review by freddi-kit — APPROVED** [review]
okay, I approved it to run on CI, let's check it

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -4,11 +4,11 @@ on:
   pull_request: {}
 jobs:
   run:
-    runs-on: macOS-latest
+    runs-on: macos-11
     name: Xcode ${{ matrix.xcode }}
     strategy:
       matrix:
-        xcode: ["12"]
```
Comment (freddi-kit) [inline .github/workflows/ci.yml:11]: Is 12.5.1 still necessary? cc: @yonaskolb

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -4,11 +4,11 @@ on:
...
+        xcode: ["12.5.1", "13.0"]
```
Comment (freddi-kit) [inline .github/workflows/ci.yml:11]: Is 12.5.1 still necessary? cc: @yonaskolb

File: CHANGELOG.md (hunk 1 above)
Comment (raptorxcz) [reply CHANGELOG.md:15]: 🤔 version 2.25.0 was not released.

Comment (raptorxcz) [reply CHANGELOG.md:15]: but maybe would be better to review of this PR https://github.com/yonaskolb/XcodeGen/pull/1130 instead of mine.

**Comment by freddi-kit** [comment]
As  @0111b said, it has a problem with Carthage, I'm making new build script for building test project with Carthage so please wait

**Comment by freddi-kit** [comment]
Ops, it looks @mthole has opened pr, I'm checking it

File: CHANGELOG.md (hunk 1 above)
Comment (freddi-kit) [reply CHANGELOG.md:15]: https://github.com/yonaskolb/XcodeGen/pull/1130#discussion_r715083051 
What i want to say is here, sorry for the few context in my comments. I already asked @mthole to do it in other PR.

**Comment by freddi-kit** [comment]
@raptorxcz Thank you for your contribution, #1130 will be merged with your commits and I think we can close this PR.

**Pull request closed without merging**